src/Sample.Extension/SampleTableBindingExtension.cs

```diff
@@ -1,96 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the MIT License. See License.txt in the project root for license information.
-
-using System;
-using System.IO;
-using System.Reflection;
-using System.Threading;
-using System.Threading.Tasks;
-using Microsoft.Azure.WebJobs;
-using Microsoft.Azure.WebJobs.Host.Bindings;
-using Microsoft.Azure.WebJobs.Host.Tables;
-using Microsoft.WindowsAzure.Storage.Table;
-
-namespace Sample.Extension
-{
-    internal class SampleTableBindingProvider : IArgumentBindingProvider<ITableArgumentBinding>
```

Comment:
Should also delete the wiki page for all this https://github.com/Azure/azure-webjobs-sdk-extensions/wiki/Binding-Customization